DevoxxClientMobile/src/main/java/com/devoxx/service/DevoxxService.java

```diff
@@ -765,9 +765,6 @@ private void retrieveExhibitionMapsInternal() {
 
     @Override
     public ObservableList<SponsorBadge> retrieveSponsorBadges() {
-        if (!isAuthenticated() && DevoxxSettings.USE_REMOTE_NOTES) {
-            throw new IllegalStateException("An authenticated user must be available when calling this method.");
-        }
         
         if (sponsorBadges == null) {
             final GluonObservableList<SponsorBadge> sponsorBadges = internalRetrieveSponsorBadges();
```

Comment:
There is a NPE here when you try to sign in as a sponsor without having an authenticated user. 
When calling `internalRetrieveSponsorBadges`, if `authenticationClient.getAuthenticatedUser()` is null, `authenticationClient.getAuthenticatedUser().getKey()` throws the NPE.